src/palindrom.js

```diff
@@ -687,23 +687,21 @@ var Palindrom = (function() {
    * @param {*} val value 
    * @param {Function} errorHandler 
    */
-  const findRangeErrors = (function createValidator(min, max) {
-    // curry validator to cache Number.MIN_SAFE_INTEGER and Number.MAX_SAFE_INTEGER for better performance, this code runs A LOT
-    return function(val, errorHandler) {
-      const type = typeof val;
-      if (type == 'object') {
-        Object.values(val).forEach(value =>
-          findRangeErrors(value, errorHandler)
-        );
-      } else if (type === 'number' && (val > max || val < min)) {
-        errorHandler(
-          new RangeError(
-            `A number that is either bigger than Number.MAX_INTEGER_VALUE or smaller than Number.MIN_INTEGER_VALUE has been encountered in a patch, value is: ${val}`
-          )
-        );
-      }
-    };
-  })(Number.MIN_SAFE_INTEGER, Number.MAX_SAFE_INTEGER);
+  function findRangeErrors(val, errorHandler) {
+    const type = typeof val;
+    if (type == 'object') {
+      Object.values(val).forEach(value => findRangeErrors(value, errorHandler));
```

Comment:
If performance is the key, then I am afraid reverting to `for .. in` is a better option :(